lab_3_keywords_textrank/main.py

```diff
@@ -173,7 +191,21 @@ def extract_pairs(tokens: tuple[int, ...], window_length: int) -> Optional[tuple
     In case of corrupt input data, None is returned:
     tokens must not be empty, window lengths must be integer, window lengths cannot be less than 2.
     """
-    pass
+    if not tokens or not isinstance(window_length, int) or window_length < 2:
+        return None
+    pairs = []
+    for num in range(len(tokens) + 1 - window_length):
+        window = tokens[num:window_length + num]
+        for element in window:
+            small = window[window.index(element) + 1:]
+            for subel in small:
+                pairs.append([element, subel])
+    unique = []
```

Comment:
again - what about set?

Reply:
I tried using set, now all works somehow... the second case I also did